Iggy_SDK_Tests/E2ETests/StreamsE2E.cs

```diff
@@ -17,93 +17,152 @@ public StreamsE2E(IggyStreamFixture fixture)
+        //     result.Should().NotBeNull();
+        //     result!.Name.Should().Be(StreamsFixtureBootstrap.UpdateStreamRequest.Name);
+        // })).ToArray();
+        // await Task.WhenAll(tasks);
     }
 
-    [Fact(Skip = SkipMessage), TestPriority(5)]
+    [Fact, TestPriority(5)]
     public async Task DeleteStream_Should_DeleteStream_Successfully()
     {
-        var tasks = _fixture.SubjectsUnderTest.Select(sut => Task.Run(async () =>
-        {
-            await sut.Invoking(async x => await x.DeleteStreamAsync(Identifier.Numeric((int)StreamsFixtureBootstrap.StreamRequest.StreamId!)))
-                .Should()
-                .NotThrowAsync();
-        })).ToArray();
-        await Task.WhenAll(tasks);
+        // act
+        await _fixture.HttpSut.Invoking(y => y
+                .DeleteStreamAsync(Identifier.Numeric((int)StreamsFixtureBootstrap.StreamRequest.StreamId!))
+            ).Should()
+            .NotThrowAsync();
+        
+        // TODO: This code block is commmented bacause TCP implementation is not working properly.
+        // var tasks = _fixture.SubjectsUnderTest.Select(sut => Task.Run(async () =>
+        // {
+        //     await sut.Invoking(async x => await x.DeleteStreamAsync(Identifier.Numeric((int)StreamsFixtureBootstrap.StreamRequest.StreamId!)))
+        //         .Should()
+        //         .NotThrowAsync();
+        // })).ToArray();
+        // await Task.WhenAll(tasks);
     }
-
-    [Fact(Skip = SkipMessage), TestPriority(6)]
+    
+    [Fact, TestPriority(6)]
     public async Task DeleteStream_Should_Throw_InvalidResponse()
     {
-        var tasks = _fixture.SubjectsUnderTest.Select(sut => Task.Run(async () =>
-        {
-            await sut.Invoking(async x => await x.DeleteStreamAsync(Identifier.Numeric((int)StreamsFixtureBootstrap.StreamRequest.StreamId!)))
-                .Should()
-                .ThrowExactlyAsync<InvalidResponseException>();
-        })).ToArray();
-        await Task.WhenAll(tasks);
+        // act
+        await _fixture.HttpSut.Invoking(y => y
+                .DeleteStreamAsync(Identifier.Numeric((int)StreamsFixtureBootstrap.StreamRequest.StreamId!))
+            ).Should()
+            .ThrowExactlyAsync<InvalidResponseException>();
+        
+        // TODO: This code block is commmented bacause TCP implementation is not working properly.
+        // var tasks = _fixture.SubjectsUnderTest.Select(sut => Task.Run(async () =>
+        // {
+        //     await sut.Invoking(async x => await x.DeleteStreamAsync(Identifier.Numeric((int)StreamsFixtureBootstrap.StreamRequest.StreamId!)))
+        //         .Should()
+        //         .ThrowExactlyAsync<InvalidResponseException>();
+        // })).ToArray();
+        // await Task.WhenAll(tasks);
     }
 
-    [Fact(Skip = SkipMessage), TestPriority(7)]
+    // [Fact(Skip = SkipMessage), TestPriority(7)]
```

Comment:
I think this can be removed already ?

Reply:
For sure!
I'll push another commit with this change!